server/.github/workflows/eslint.yml

```
@@ -0,0 +1,37 @@
+      - name: Install pnpm package manager
+        uses: pnpm/action-setup@v4
+        with:
+          version: 10
+
+      # Paso 3: Instala las dependencias del proyecto
+      - name: Install dependencies
+        run: pnpm install --dev
+
+      # Paso 4: Ejecuta ESLint usando el comando definido en package.json
+      - name: Run ESLint
+        run: pnpm lint
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)